client/my-sites/plans-features-main/style.scss

```diff
@@ -13,11 +13,20 @@
 			display: flex;
 			flex-flow: row wrap;
 
+			@include breakpoint( '>660px' ) {
+				flex-flow: row;
```

Comment:
```suggestion
				flex-flow: row;
				max-width: 1000px;
```

Reply:
Oops, that works much better - fixed, thank you! :)